examples/plot_mtw.py

```diff
@@ -0,0 +1,237 @@
+"""
+Multi-subject joint source localization with MTW
+================================================
+
+The aim of this tutorial is to show how to leverage functional similarity
+across subjects to improve source localization with a spatial prior using
+Optimal transport. Unlike multi-task Lasso which assumes that the exact same
+sources are active for all subjects, MTW promotes a soft spatial proximity
+prior of sources across subjects. This example illustrates this on the
+the high frequency SEF MEG dataset of (Nurminen et al., 2017) which provides
+MEG and MRI data for two subjects.
+"""
+
+# Author: Hicham Janati (hicham.janati@inria.fr)
+#
+# License: BSD (3-clause)
+
+import mne
+import os
+import os.path as op
+from mne.parallel import parallel_func
+from mne.datasets import hf_sef
+from matplotlib import pyplot as plt
+
+from groupmne import compute_group_inverse, prepare_fwds, compute_fwd
+
+##########################################################
+# Download and process MEG data
+# -----------------------------
+#
+# For this example, we use the HF somatosensory dataset [2].
+# We need the raw data to estimate the noise covariance
+# since only average MEG data (and MRI) are provided in "evoked".
+# The data will be downloaded in the same location
+
+
+_ = hf_sef.data_path("raw")
+data_path = hf_sef.data_path("evoked")
+meg_path = data_path + "/MEG/"
+
+data_path = op.expanduser(data_path)
+subjects_dir = data_path + "/subjects/"
+os.environ['SUBJECTS_DIR'] = subjects_dir
+
+raw_name_s = [meg_path + s for s in ["subject_a/sef_right_raw.fif",
+              "subject_b/hf_sef_15min_raw.fif"]]
+
+
+def process_meg(raw_name):
+    """Extract epochs from a raw fif file.
+
+    Parameters
+    ----------
+    raw_name: str.
+        path to the raw fif file.
+
+    Returns
+    -------
+    epochs: Epochs instance
+
+    """
+    raw = mne.io.read_raw_fif(raw_name)
+    events = mne.find_events(raw)
+
+    event_id = dict(hf=1)  # event trigger and conditions
+    tmin = -0.05  # start of each epoch (50ms before the trigger)
+    tmax = 0.3  # end of each epoch (300ms after the trigger)
+    baseline = (None, 0)  # means from the first instant to t = 0
+    epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True,
+                        baseline=baseline)
+    return epochs
+
+
+epochs_s = [process_meg(raw_name) for raw_name in raw_name_s]
+evokeds = [ep.average() for ep in epochs_s]
+
+# compute noise covariance (takes a few minutes)
```

Comment:
really takes a few mins with just 100 epochs? rather than saying this say that you use only 100 epochs to save time.

Reply:
you're right it doesn't, it was an old comment